src/components/input/RangeInput.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div ref="root" class="range" :class="{ active: isActive }" @click="onRootClick">
+    <location-input class="location" :isDisabled="!isActive" v-model="location" />
+    <transport-type class="transport-type" :isDisabled="!isActive" v-model="transportType" />
+    <travel-time class="travel-time" :isDisabled="!isActive" v-model="travelTime" />
+  </div>
+</template>
+<style scoped lang="scss">
+@import "../../style/variables";
+
+.range {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 48px;
+  background-color: white;
+  height: 76px;
+  cursor: pointer;
+  /*transition: height 0.5s ease-in-out, background-color 0.5s ease-in-out;*/
```

Comment:
Forgot to remove?

Reply:
ah yes, this was future work in progress :D